samples/client/petstore/powershell_test/src/IO.Swagger/API/PetApi.ps1

```diff
@@ -0,0 +1,164 @@
+function PetApi-addPet {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = , ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
```

Comment:
I tried to fix it and now the position starts at 1. Is that ok or it must be started at 0?